### ISSUE TITLE
Invalid magic bytes in file header

### ISSUE THREAD

**Issue by ReenigneArcher** [linuxdeploy/linuxdeploy#185]
Attempting to use linuxdeploy in github CLI and getting an error about invalid magic bytes in file header.

```txt
-- Deploying executables -- 
Deploying executable ./sunshine 
WARNING: Could not find copyright files for file ./sunshine using dpkg-query 
terminate called after throwing an instance of 'linuxdeploy::core::elf_file::ElfFileParseError'
  what():  Invalid magic bytes in file header
/home/runner/work/_temp/6ca18cef-ad31-4702-b6a8-9daabbe0a662.sh: line 6: 12149 Aborted                 (core dumped) ./linuxdeploy-x86_64.AppImage --appdir AppDir -e ./sunshine -i "$ICON_FILE" -d "$DESKTOP_FILE" --output appimage
Error: Process completed with exit code 134.
```

Here's the full run:
https://github.com/ReenigneArcher/sunshine-AppImage/runs/4347810623?check_suite_focus=true

I have run the same code in an Ubuntu 20.04 VM and it creates the AppImage without issue. Any idea how to solve?

**Comment by ReenigneArcher** [comment]
Never mind, this was an issue with the current working directory

**Comment by TheAssassin** [comment]
Mind to tell whether the error was shown despite the file was missing? In that case, I'd like to add a proper error message.

**Comment by ReenigneArcher** [comment]
I'm not sure which file it was complaining about specifically... whether it was the executable, icon, or desktop file. Likely all 3 were off.

Here's the full log (https://github.com/ReenigneArcher/sunshine-AppImage/runs/4347810623?check_suite_focus=true)... I think you should be able to review.

**Comment by TheAssassin** [comment]
The log doesn't really tell whether the file was found or not, as you noted in your first post, linuxdeploy complains about some invalid file header. Anyway, I'll check whether a test case needs to be added for this.

**Comment by ReenigneArcher** [comment]
So for the failed runs, I left the working directory as the build directory and I think changing to a different "step" of the workflow, it moves back to the original directory automatically (not sure if that's really the case). Before exiting the build step, I moved back to the original directory.

Then in the make appimage step, I adjusted the paths of the binary, icon, and desktop file. The paths were the real change, so I'm pretty certain the file(s) weren't found.

https://github.com/ReenigneArcher/sunshine-AppImage/commit/d1afc5b894395daf9851de775384d59219f73095#diff-311182179a449e6a94a137f94cf675268010e98c5ab6df536651aa6fcaa69556
